specs/Ihec_metadata_specification.md

```diff
@@ -1,70 +1,64 @@
 
-The same attribute may be used multiple times in a single XML record. This may be most useful for supplying URIs to multiple ontologies or for supplying multiple references to a single ontology such as in the case of DISEASE_ONTOLOGY_URI.
+The same attribute may be used multiple times in a single XML record. This may be most useful, for example, for supplying URIs to multiple ontologies or for supplying multiple references to a single ontology such as in the case of DISEASE_ONTOLOGY_URI. For example, describing a brain primary tissue using ontology terms for ('Brodmann (1909) area 8', 'Brodmann (1909) area 9') would be:
 
-Documentation for the core SRA XML elements is here:
-
-http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=doc
-
-The SRA XML schema http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=docas are here:
-
-* http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=xml_schemas
-
-Tags with controlled vocabularies are labelled as “Controlled Vocabulary”
-
-Tags with ontolgies are labelled as “Ontology”
+```
+<SAMPLE_ATTRIBUTE>
+    <TAG>SAMPLE_ONTOLOGY_URI</TAG>
+    <VALUE>http://purl.obolibrary.org/obo/UBERON_0013539</VALUE>
+</SAMPLE_ATTRIBUTE>
+<SAMPLE_ATTRIBUTE>
+    <TAG>SAMPLE_ONTOLOGY_URI</TAG>
+    <VALUE>http://purl.obolibrary.org/obo/UBERON_0013540</VALUE>
+</SAMPLE_ATTRIBUTE>
+```
 
 ## Ontologies
 
 __Only__ terms from following ontologies are acceptable for annotating the metadata:
 
-##### Sample Ontologies
-
-* Cell Lines - Experimental Factor Ontology (EFO)
+Field __SAMPLE_ONTOLOGY_URI__:
+* Cell Lines: Experimental Factor Ontology (__EFO__ - [https://www.ebi.ac.uk/efo/](https://www.ebi.ac.uk/efo/))
+* Primary Cells: Cell Ontology (__CL__ - [http://cellontology.org](http://cellontology.org))
+* Primary Tissue: Uberon (__UBERON__ - [http://uberon.org](http://uberon.org))
 
-* Primary Cells - Cell Ontology (CL)
+Fields __DISEASE_ONTOLOGY_URI__ and __DONOR_HEALTH_STATUS_ONTOLOGY_URI__:
+* NCI Metathesaurus (__NCIM__ - [https://ncim.nci.nih.gov/ncimbrowser/](https://ncim.nci.nih.gov/ncimbrowser/))
 
-* Primary Tissue - Uberon
+Field __EXPERIMENT_ONTOLOGY_URI__:
+* Ontology for Biomedical Investigations (__OBI__ - [http://obi-ontology.org/](http://obi-ontology.org/))
 
-##### Disease Ontologies
+Field __MOLECULE_ONTOLOGY_URI__:
+* Sequence Ontology (__SO__ - [http://www.sequenceontology.org/](http://www.sequenceontology.org/)) 
 
-* Disease - NCI Metathesaurus
+Tags with controlled vocabularies are labelled as "Controlled Vocabulary".
 
-##### Experiment Ontologies
+Tags with ontologies are labelled as "Ontology".
 
-* Assays and Platforms - Ontology for Biomedical Investigations (OBI)
-
-# IHEC Extensions 
-
-## Level 0 Data
-
-### SAMPLES
-
-#### Cell Line
-
-__MOLECULE__ - (Controlled Vocabulary) The type of molecule that was extracted from the biological material. Include one of the following: total RNA, polyA RNA, cytoplasmic RNA, nuclear RNA, genomic DNA, protein, or other.
+# SAMPLES
 
```

Comment:
We need a note for metadata resubmissions here too. Something that reads like:

###### Note for metadata resubmission
In order to pass IHEC metadata validation, all datasets submitted prior to 2018 __must__ include all the following sample properties, defined within each specific BIOMATERIAL_TYPE below.

Reply:
Done.